API.md

```diff
@@ -459,6 +460,25 @@ Runs the given function with the supplied value, and then resolves with that val
 tapP(a => Promise.resolve('b'), 'a') //=> Promise 'a'
 ```
 
+### useWithP
+
+`@articulate/funky/lib/useWithP`
+
+```haskell
+useWithP :: (b -> c -> Promise d) -> [(a -> Promise b), (a -> Promise c)] -> a -> Promise d
+```
+
+An async version of [`R.useWith`](https://ramdajs.com/docs/#useWith) that accepts Promise-returning transformer and converging functions.
```

Comment:
Since it's not `converge`, it's not really a "converging function."  Maybe "original function"?